.github/workflows/diffhawk.yml

```
@@ -46,11 +52,11 @@ jobs:
           repository: amalshaji/diffhawk
           path: .github/actions/diffhawk
           token: ${{ secrets.GH_PAT }}
-      
+
       # Use it locally
       - uses: ./.github/actions/diffhawk
         with:
-          github_token: ${{ secrets.GH_PAT }}
+          github_token: ${{ steps.get-token.outputs.token }}
```

actionlint (1.7.9)

property "get-token" is not defined in object type {}

(expression)